<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>@elizeubarbosaabreu</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerador de QR-Code</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Digite seu texto ou link...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Gerar QR CODE</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Salvar QR CODE &gt;&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
